tests/test.py

```diff
@@ -25,3 +29,33 @@ def test_deprecated_has_version():
     #    https://github.com/pypa/setuptools/issues/1299
 
     assert 'Legacy' not in version.__class__.__name__
+
+
+@pass_through
+def file_in_stack():
```

Comment:
Looks like the py34 test environment has the same problem as I'm seeing on my system. [test output](https://ci.appveyor.com/project/tantale/deprecated/builds/28725583) It was the only test environment that replicated my problem in [my original pull request test](https://ci.appveyor.com/project/tantale/deprecated/builds/28705135). I'm not sure what is different about that environment.

py34 replicates the problem in the pass_through test:
https://ci.appveyor.com/project/tantale/deprecated/builds/28725583/job/w3a7smf17v5awg2h#L161

py35 (or any other) does not replicate the problem in the pass_through test:
https://ci.appveyor.com/project/tantale/deprecated/builds/28725583/job/jv1wm6n7x3b30fcg#L156

I forced the new tests to always fail so we could compare the tracebacks. Here is the py34 stack reported reported when using the pass_through decorator.
```python
E           File "C:\projects\deprecated\tests\test.py", line 46, in test_file_in_wrapt_stack
E             file_in_stack()
E           File "c:\projects\deprecated\.tox\py34\lib\site-packages\wrapt\wrappers.py", line 564, in __call__
E             args, kwargs)
E           File "C:\projects\deprecated\tests\wrapt_decorators.py", line 8, in pass_through
E             return wrapped(*args, **kwargs)
E           File "C:\projects\deprecated\tests\test.py", line 36, in file_in_stack
E             stack = traceback.format_stack(limit=10)
```

The py35 output:
```python
E           File "c:\projects\deprecated\.tox\py35\lib\site-packages\_pytest\python.py", line 170, in pytest_pyfunc_call
E             result = testfunction(**testargs)
E           File "C:\projects\deprecated\tests\test.py", line 46, in test_file_in_wrapt_stack
E             file_in_stack()
E           File "C:\projects\deprecated\tests\wrapt_decorators.py", line 8, in pass_through
E             return wrapped(*args, **kwargs)
E           File "C:\projects\deprecated\tests\test.py", line 36, in file_in_stack
E             stack = traceback.format_stack(limit=10)
```

I also added a similar unit test using the deprecated decorator.